crates/kernel/src/builtins/bf_properties.rs

```diff
@@ -204,7 +204,7 @@ fn bf_add_property(bf_args: &mut BfCallState<'_>) -> Result<BfRet, BfErr> {
             *location,
             *location,
             name.as_str(),
-            bf_args.caller_perms(),
+            attrs.owner.unwrap(),
```

Comment:
This kind of stuff will be more fully tested when I port https://github.com/toddsundsted/stunt/blob/master/test/test_objects_and_properties.rb; this is enough to make the `recycle` suite pass.